packages/foundry/src/cloneProjectTemplate.ts

```diff
@@ -0,0 +1,9 @@
+import degit from "degit";
+import fse from "fs-extra";
```

Comment:
import { ensureDir } from "fs-extra/esm";

There is an ESM export that allows you to import methods directly. It probably help with treeshaking as well. It only works for methods that are directly created in fs-extra.

https://www.npmjs.com/package/fs-extra


> ESM
> There is also an fs-extra/esm import, that supports both default and named exports. However, note that fs methods are not included in fs-extra/esm; you still need to import fs and/or fs/promises seperately:

```
import { readFileSync } from 'fs'
import { readFile } from 'fs/promises'
import { outputFile, outputFileSync } from 'fs-extra/esm'

```

For ther other place were we use fs-extra, i would change it to

```
import { readFile, writeFile } from "node:fs/promises";
```